www-apps/phpsysinfo/phpsysinfo-3.2.10-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp
+
+DESCRIPTION="A customizable PHP script that displays information about your system nicely"
+HOMEPAGE="https://rk4an.github.com/phpsysinfo/"
+SRC_URI="https://github.com/rk4an/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~hppa ~x86"
```

Comment:
Have you tested the package on hppa yourself? If not, you should file a KEYWORDREQ for it rather than assuming that it works (see https://devmanual.gentoo.org/keywording/index.html )

Re-keywording after a keyword drop is like keywording a new package.

Reply:
I see, changed, thanks.